src/main/java/org/benetech/servicenet/conflict/detector/OrganizationConflictDetector.java

```diff
@@ -15,15 +15,16 @@
     @Override
     public List<Conflict> detectConflicts(Organization current, Organization offered) {
         List<Conflict> conflicts = new LinkedList<>();
-
         conflicts.addAll(detectConflicts(current, offered, current.getName(), offered.getName(), "name"));
         conflicts.addAll(detectConflicts(current, offered, current.getAlternateName(), offered.getAlternateName(),
             "alternateName"));
         conflicts.addAll(detectConflicts(current, offered, current.getDescription(), offered.getDescription(),
             "description"));
         conflicts.addAll(detectConflicts(current, offered, current.getEmail(), offered.getEmail(), "email"));
-        conflicts.addAll(detectConflicts(current, offered, UrlNormalizationUtils.normalize(current.getUrl()),
-            UrlNormalizationUtils.normalize(offered.getUrl()), "url"));
+        if (detect(UrlNormalizationUtils.normalize(getString(current.getUrl())),
+            UrlNormalizationUtils.normalize(getString(offered.getUrl())))) {
+            conflicts.addAll(detectConflicts(current, offered, current.getUrl(), offered.getUrl(), "url"));
```

Comment:
it seems like you could follow a convention in this class (like it was before), and just handle nulls in UrlNormalizationUtils by returning the url if it's null there. then the detectConflicts method should handle the rest. Could try it out?

Reply:
Approach that was used before gives a normalized URL in displayed conflict entity table. I think that we shouldn't change data on ServiceNet and they should be displayed in original form, not normalized.

Reply:
all that needs to be changed here is to add a null check in UrlNormalizationUtils